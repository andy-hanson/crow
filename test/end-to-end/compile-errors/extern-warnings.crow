main void()
	()

f void() bare, c extern

f void() unsafe, c extern

f void() summon, c extern

f void() pure
	()
