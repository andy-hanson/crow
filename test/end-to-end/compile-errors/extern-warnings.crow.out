exit code: 0
stderr:
test/end-to-end/compile-errors/extern-warnings.crow 4:1-4:2 'bare' is redundant given 'extern'.
test/end-to-end/compile-errors/extern-warnings.crow 6:1-6:2 'unsafe' is redundant given 'extern'.
test/end-to-end/compile-errors/extern-warnings.crow 8:1-8:2 'summon' is redundant given 'extern'.
test/end-to-end/compile-errors/extern-warnings.crow 10:1-10:2 A function is already 'pure' by default.
